Bottom: 1dc720c20d05beab39231824f38fafc5f03d27cb
Top:    62cf049d1dd65a19827bd47aa8c4b563aab2befe
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-09 13:26:07 -0400

Refresh of stac9766-codec

---

diff --git a/sound/soc/codecs/stac9766.c b/sound/soc/codecs/stac9766.c
index 9d85f34..b32b7bf 100644
--- a/sound/soc/codecs/stac9766.c
+++ b/sound/soc/codecs/stac9766.c
@@ -88,7 +88,7 @@ static const struct snd_kcontrol_new stac9766_snd_ac97_controls[] = {
 	SOC_SINGLE("Mono Out Volume", AC97_MASTER_MONO, 0, 31, 1),
 	SOC_SINGLE("Mono Out Switch", AC97_MASTER_MONO, 15, 1, 1),
 
-	SOC_DOUBLE("Record Volume", AC97_REC_GAIN, 8, 0, 31, 1),
+	SOC_DOUBLE("Record Volume", AC97_REC_GAIN, 8, 0, 15, 1),
 	SOC_SINGLE("Record Switch", AC97_REC_GAIN, 15, 1, 1),
